src/Popover/Popover.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { ListOption, PopoverList } from './Popover.styles';
+
+export interface Props {
+  variant?: string;
+  disabled?: boolean;
+  children?: {
```

Comment:
I don't think this should be optional? But we can think more and address it later.

Reply:
I agree that it would be weird to have the popover list with no children in it, but I was more following React practice of children not being mandatory. appy for either way

Reply:
Up to you in that case, my thought is a child should be mandatory if it is key to the functionality of something, same as any other prop really. But I am not religious about it.